src/screens/SearchBeers/SearchBeers.jsx

```diff
@@ -168,7 +168,7 @@ function SearchBeersComponent() {
   };
 
   async function getBeersByName() {
-    const beers = await fetchBeers([`beer_name=${params.beerName}`]);
+    const beers = await fetchBeers([`beer_name=${params.beerName}`]) || [];
```

Comment:
Do you still need this `|| []` after implementing required condition?

Reply:
Fixed !